DER/ExchangeTradedDerivatives/ExchangeTradedOptions.rdf

```diff
@@ -98,54 +103,50 @@
 		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/SEC/Equities/EquityInstruments/"/>
 		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/SEC/Securities/SecuritiesListings/"/>
 		<owl:versionIRI rdf:resource="https://spec.edmcouncil.org/fibo/ontology/DER/ExchangeTradedDerivatives/ExchangeTradedOptions/"/>
 		<fibo-fnd-utl-av:hasMaturityLevel rdf:resource="&fibo-fnd-utl-av;Provisional"/>
 	</owl:Ontology>
 	
+	<owl:Class rdf:about="&fibo-der-der-opt;OptionOnFuture">
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-dbt-ex;hasExerciseTerms"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-etd-eto;FutureOptionsExerciseTerms"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+	</owl:Class>
+	
 	<owl:Class rdf:about="&fibo-der-etd-eto;BondOption">
-		<rdfs:subClassOf rdf:resource="&fibo-der-etd-eto;OptionContractOnFutures"/>
+		<rdfs:subClassOf rdf:resource="&fibo-der-der-opt;OptionOnFuture"/>
 		<rdfs:subClassOf rdf:resource="&fibo-der-sbd-sbd;DebtInstrumentDerivative"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-der-drc-bsc;hasUnderlier"/>
-				<owl:someValuesFrom rdf:resource="&fibo-der-etd-eto;BondOptionUnderlying"/>
+				<owl:someValuesFrom rdf:resource="&fibo-sec-dbt-bnd;Bond"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label xml:lang="en">bond option</rdfs:label>
-		<skos:definition xml:lang="en">An option on a traded futures contract for a Bond. When exercised, the holder exercises the right to buy the future on a Bond, s/he does not exercise the right to buy a bond.</skos:definition>
-		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">For example an option on a Treasury Bill is an option on a T Bill Future, not on a T Bill itself. Hence these are defined as a kind of futures option.</fibo-fnd-utl-av:explanatoryNote>
-	</owl:Class>
-	
-	<owl:Class rdf:about="&fibo-der-etd-eto;BondOptionUnderlying">
-		<rdfs:subClassOf rdf:resource="&fibo-der-etd-eto;OptionOnFuturesUnderlying"/>
-		<rdfs:label xml:lang="en">bond option underlying</rdfs:label>
-		<skos:definition xml:lang="en">The underlying on a Bond Option, which is the corresponding Bond Future contract on the same exchange.</skos:definition>
-		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">This is the underlying in the sense that when the option is exercised the investor takes delivery of the corresponding futures contract.</fibo-fnd-utl-av:explanatoryNote>
+		<skos:definition xml:lang="en">option on a futures contract for a bond</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">When exercised, the holder exercises the right to buy the future on a Bond, s/he does not exercise the right to buy a bond. For example an option on a Treasury Bill is an option on a T Bill Future, not on a T Bill itself. Hence these are defined as a kind of futures option.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
not very clear. And I think "a" in the phase "option on a Treasury Bill" seems wrong.